### DISCUSSION TITLE
column "fts" can only be updated to DEFAULT

### DISCUSSION THREAD

**Discussion by Electric-Dragon** [supabase/supabase#5902] [Questions] [answered]
I followed [this](https://supabase.com/docs/guides/database/full-text-search#searchable-columns) guide in the official Supabase documentation to create a column for full-text search. This code was used to create the full-text search column

```
alter table 
  series 
add column 
  fts tsvector generated always as (to_tsvector('english', summary || ' ' || title)) stored;
  
create index series_fts on series using gin (fts);

select id, fts 
from series;
```

 However, when I update a column unrelated to the `summary` and `title` (I am updating the `status` column), I get this error `column "fts" can only be updated to DEFAULT`

**Comment by Electric-Dragon** [comment]
Actually then I was changing the status from the dashboard...but now that I tried it out through `JS`, it did not fail. Can someone let me know why this happened?

**Comment by GaryAustin1** [comment, marked as answer]
The dashboard currently tries to update the entire row (all columns) with new or existing data.  A fix was made last week to only update the column you changed, but I assume that is not deployed yet.

**Reply by Electric-Dragon** [reply]
So it's a dashboard issue and nothing related to my backend?

**Reply by GaryAustin1** [reply]
Yes, especially if it worked for you thru an api call.  You can't update "calculated" columns.
